articles/install-guide/index.md

```diff
@@ -1,40 +1,89 @@
-<div class="nextstepaction">
-[Install on Visual Studio 2017 (Windows 10)](xref:microsoft.quantum.install.vs-2017)
-</div>
-</td>
+Be mindful that all changes you do to notebooks launched this way cannot be saved. In order 
+to create new notebooks or write new quantum applications you will need to install 
+IQ# to your local machine.
 
-<td>
-[<img src="../media/install-guide-select-vs-code.png" width="90%">](xref:microsoft.quantum.install.vs-code)
-<div class="nextstepaction">
-[Install on Visual Studio Code (Windows 10, macOS, or Linux)](xref:microsoft.quantum.install.vs-code)
-</div>
-</td>
-</tr>
-</table>
+
+## Installing IQ# ##
+
+IQ# (pronounced i-q-sharp) provides the core functionality of compiling and simulating Q# operations.
+Installing IQ# on your machine typically takes less than 10 minutes; just follow these two steps:
+
+1. Install the latest version of [.NET Core SDK](https://dotnet.microsoft.com/) (2.1 or later) by 
+  following the instructions from the [.NET downloads page](https://www.microsoft.com/net/download).
+2. From the command line, execute:
+   ```Command Prompt
+   dotnet tool install -g Microsoft.Quantum.IQSharp
+   ```
+
+Your installation of the Quantum Development Kit is now complete!
+
+To verify IQ# was correctly installed, from the command line type: `dotnet iqsharp --version`. You should see the IQ# version reported on the output, for example:
+```
+C:\>dotnet iqsharp --version
+Language kernel: 0.5.1903.2501
+Jupyter core: 1.1.12077.0
+```
 
 > [!NOTE]
-> The Quantum Development Kit requires a 64-bit installation of Windows 10, macOS, or Linux.
-> Microsoft's quantum simulator, installed with the Quantum Development Kit, utilizes Advance Vector Extensions (AVX), and thus can run significantly more efficiently on an AVX-enabled CPU.
+> IQ# requires a 64-bit installation of Windows 10, macOS, or Linux.
+>
+> Microsoft's quantum simulator, installed with the Quantum Development Kit, utilizes Advance Vector Extensions (AVX), 
+> and thus can run significantly more efficiently on an AVX-enabled CPU.
 > The Quantum Development Kit will still run on non–AVX enabled CPUs, but may not be as efficient.
 > Intel processors shipped in Q1 2011 (Sandy Bridge) or later support AVX.
+>
+> Microsoft's quantum simulator utilizes OpenMP at runtime, on Linux systems you might need to manually install `libgomp`.
+
+
+## Classical Hosts ##
+
+Applications developed with Microsoft's Quantum Development Kit typically consists of two parts:
+1. One or more quantum algorithms, implemented using the Q# quantum programming language.
+2. A classical program, implemented in a classical programming language like Python or C#, 
+  that serves as the main entry point and will invoke Q# operations 
+  when it wants to execute a quantum algorithm.
+
+Once you have installed IQ#, these articles explain how to enable interoperability between Q# and other classical languages:
+
+* [C# getting started](xref:microsoft.quantum.install.csharp)
```

Comment:
iQSharp is not required for C#, is it?

Reply:
Talked about this. Using this as a way to verify that Q# is ready to go and to give people something to install.